server/fleet/service.go

```diff
@@ -64,8 +64,12 @@ type Service interface {
 
 	// AuthenticateOrbitHost loads host identified by orbit's nodeKey. Returns an error if that nodeKey doesn't exist
 	AuthenticateOrbitHost(ctx context.Context, nodeKey string) (host *Host, debug bool, err error)
-	// EnrollOrbit enrolls orbit to Fleet by using the enrollSecret and returns the orbitNodeKey if successful
-	EnrollOrbit(ctx context.Context, hardwareUUID, hardwareSerial, enrollSecret string) (orbitNodeKey string, err error)
+	// EnrollOrbit enrolls an orbit instance to Fleet by using the the host information + enroll secret
```

Comment:
Typo: `the the` --> `the`

Reply:
[Fix typo](https://github.com/fleetdm/fleet/pull/10308/commits/ee0418aa112a7b25706920398bafac1ab60e26ba).